packages/admin-ui/src/lib/core/src/components/main-nav/main-nav.component.html

```diff
@@ -24,6 +24,7 @@
                                     [attr.data-item-id]="section.id"
                                     [routerLink]="getRouterLink(item)"
                                     routerLinkActive="active"
+                                    (click)="item.onClick && item.onClick()"
```

Comment:
We'd need to pass the click event here to satisfy the type of the `onClick` function:
```
item.onClick($event)
```